packages/unmock-core/src/__tests__/generator.test.ts

```diff
@@ -1,140 +1,164 @@
-import * as path from "path";
-import NodeBackend from "../backend/index";
-import { responseCreatorFactory } from "../generator";
+import {
+  matchUrls,
+  prunePathItem,
+  matches,
+  refName,
+  firstElementOptional,
+  keyLens,
+  operationOptional,
+  useIfHeader,
+  identityGetter,
+  hoistTransformer,
+} from "../generator";
+import { PathItem, OpenAPIObject } from "loas3/dist/generated/full";
+import { some, none } from "fp-ts/lib/Option";
+import { ISerializedRequest } from "../interfaces";
 
-const mockOptions = {
-  flaky: () => false,
-  isWhitelisted: (_: string) => false,
-  log: (_: string) => undefined,
-  useInProduction: () => false,
-};
+test("hasUrl works", () => {
+  expect(matchUrls(
```

Comment:
I think it would be nice if each of the expects was a separate test under a common `describe()`, makes it easier to see what is actually being tested in each case. But it's not important now, just a general comment.